src/main/java/de/muenchen/isi/infrastructure/entity/Bauvorhaben.java

```diff
@@ -28,7 +28,7 @@
 @EqualsAndHashCode(callSuper = true)
 public class Bauvorhaben extends BaseEntity {
 
-    @Column(nullable = false)
+    @Column(nullable = false, unique = true)
```

Comment:
Die im Repository mittels findBy nach dem Namen direkt in der Datenbank gesucht wird und wegen eines fehlenden Suchindexes diese Suche sequentiell durchgeüfrht wird, sollte für das Attribute nameAbfrage ein Suchindex erstellt definiert werden.

Somit wird bei vielen Entitäten die Suche erheblich beschleunigt.

```java
@Table(
        indexes = {
                @Index(name = "name_vorhaben_index", columnList = "nameVorhaben")
        }
)
```